Update fileio.rst (#980)
ansys/pyprimemesh: doc/source/user_guide/fileio.rst

.. code-block:: python

    mesh_util = prime.lucid.Mesh(model=model)
    mesh_util.read(file_name=mixing_elbow, cad_reader_route=prime.CadReaderRoute.DISCOVERY)

> Four CAD import routes are available in PyPrimeMesh:

Reviewer: ```suggestion
Five CAD import routes are available in PyPrimeMesh:
```

   Windows platform supports importing files using the Discovery reader.
   Ensure to install Discovery for Discovery reader. 

 ..note:

>       When you import SpaceClaim or Discovery route, your defined inputs in the imported files are ignored. 
>       You may need to import CAD  in the application stand-alone with the options defined and export it to 
>       a native format(fmd or pmdb) for Ansys PrimeServer.

Reviewer: ```suggestion
```suggestion
      When you import CAD using the SpaceClaim or Discovery reader routes the user import options in those
      applications are ignored.
      If the options are required then you can import CAD in the application stand-alone with the options defined and
      export it to a native format (FMD or PMDB) to import into Ansys Prime Server.
```
```

 * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.
     Ensure to install Workbench for Workbench reader. 
    
To view the CAD files supported for the Workbench route on different platforms, see
`CAD Support <https://www.ansys.com/it-solutions/platform-support>`_ on the Ansys website. 

.. note::
    * Program controlled supports faceted data. Workbench supports BRep geometry(non-faceted) data. 
      Discovery or SpaceClaim supports both BRep geometry and faceted data.

>     * When deploying scripts Workbench CAD reader routes, ensure that the CAD configuration
>       are consistent in the deployed environment for Workbench.

Reviewer: ```suggestion
    * When deploying scripts using the Workbench CAD reader route, ensure that the user options for the installed
       application are consistent in the deployed environment.
    * When deploying scripts using the CAD configuration, ensure that the configuration is consistent in the deployed
      environment.
```

   Windows platform supports importing files using the Discovery reader.
   Ensure to install Discovery for Discovery reader. 

> .. note:
>     * When you import CAD using the SpaceClaim or Discovery reader routes the user import options in those applications are ignored.
>     
>     * If the options are required then you can import CAD in the application stand-alone with the options defined and export it to a native format (FMD or PMDB) to import into Ansys Prime Server.
> 
>  * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.
>      Ensure to install Workbench for Workbench reader. 

Reviewer: ```suggestion
 * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.
     Ensure to install Workbench for Workbench reader.

.. note:
    * When you import CAD using the SpaceClaim or Discovery reader routes the user import options in those applications are ignored.
    
    * If the options are required then you can import CAD in the application stand-alone with the options defined and export it to a native format (FMD or PMDB) to import into Ansys Prime Server.
```

   Windows platform supports importing files using the Discovery reader.
   Ensure to install Discovery for Discovery reader. 

>  ..note:
> ```suggestion
>       When you import CAD using the SpaceClaim or Discovery reader routes the user import options in those
>       applications are ignored.
>       If the options are required then you can import CAD in the application stand-alone with the options defined and
>       export it to a native format (FMD or PMDB) to import into Ansys Prime Server.
> 

Reviewer: ```suggestion
.. note:
    * When you import CAD using the SpaceClaim or Discovery reader routes the user import options in those applications are ignored.
    
    * If the options are required then you can import CAD in the application stand-alone with the options defined and export it to a native format (FMD or PMDB) to import into Ansys Prime Server.

```
